lib/v2/hebeu/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/kexin/:category', require('./kexin'));
```

Comment:
Since you set a fallback value for `category`, it will be an optional parameter instead of required parameter.

https://github.com/DIYgod/RSSHub/blob/92d9c57cbe02e9deb6298642fbeba1616573a01a/lib/v2/hebeu/kexin.js#L16

```suggestion
    router.get('/kexin/:category?', require('./kexin'));
```